src/Installers/Windows/AspNetCoreModule-Setup/build/settings/common.props

```diff
@@ -25,7 +25,9 @@
 
   <ItemDefinitionGroup>
     <ClCompile>
-      <WarningLevel>Level3</WarningLevel>
+      <!-- Microsoft.Security.SystemsADM.10086 - SDL required warnings -->
+      <WarningLevel>Level4</WarningLevel>
+      <TreatSpecificWarningsAsErrors>4018;4055;4146;4242;4244;4267;4302;4308;4509;4510;4532;4533;4610;4611;4700;4701;4703;4789;4995;4996</TreatSpecificWarningsAsErrors>
```

Comment:
Add a comment about what these are? No need to enumerate all of them, maybe just a link to a doc/whatever lists the requirement

Reply:
The comment above covers that. Searching for that string publicly pulls up the relevant GitHub issue and internally takes you to the exact requirements.

Reply:
Added link to issue too.